src/Visualization/Python/PlotControlSystem.py

```diff
@@ -194,6 +194,8 @@ def extract_relevant_columns(
     # Plot all components for all systems. The name for the legend differs
     # slightly from the name in the DataFrame, just to make it prettier for the
     # user. Since both figures will have the same lines, we only show one legend
+    line_styles = ["-", "--", ":", "-."]
+    line_style_cycler = 0
     for system in control_system_components:
```

Comment:
Instead of the cycler, you can do
```py
line_styles = ["-", "--", ":", "-."]
for i, system in enumerate(control_system_components):
    ...
    linestyle=linestyles[ i % len(line_styles)]
```